libs/processor_core.py

```diff
@@ -0,0 +1,93 @@
+from threading import Thread
+from queue import Queue
+from multiprocessing.managers import BaseManager
+import logging
+from share.commands import Commands
+from libs.distancing import Distancing as CvEngine
+
+logger = logging.getLogger(__name__)
+
+class QueueManager(BaseManager): pass
+
+
+class ProcessorCore:
+
+    def __init__(self, config):
+        self.config = config 
+        self._cmd_queue = Queue()
+        self._result_queue = Queue()
+        self._setup_queues()
+        self._engine = CvEngine(self.config)
+        self._tasks = {}
+       
+
+    def _setup_queues(self):
+        QueueManager.register('get_cmd_queue', callable=lambda: self._cmd_queue)
+        QueueManager.register('get_result_queue', callable=lambda: self._result_queue)
+        self._host = self.config.get_section_dict("CORE")["Host"]
+        self._queue_port = int(self.config.get_section_dict("CORE")["QueuePort"])
+        auth_key = self.config.get_section_dict("CORE")["QueueAuthKey"]
+        self._queue_manager = QueueManager(address=(self._host, self._queue_port), authkey=auth_key.encode('ascii'))
```

Comment:
What about using 'utf-8'?

Reply:
Oh sure, as the whole idea of how to properly save and read the auth_key is not decided I just implemented a basic idea. Maybe it is better to just keep the binary or encoded or hashed or whatever - I am not experienced with authentication concerns.